dev-manuals/architecture/general-architure.md

```diff
@@ -0,0 +1,34 @@
+# General architecture
+
+## Component Structure
+
+![component diagram](diagrams/gamify-it-architecture.svg)
+
+In general, _Gamify-IT_ can be split up into three main components: the _Overworld_, the _Lecturer Interface_, and the _Minigames_.  
+Purpose:
+
+- minigame: A small game to learn something by playing
+- overworld: consists of areas you can walk around in, interact with your environment or start minigames
+- lecturer interface: The allmighty configurator to configure what happens where in the overworld, or to set what is displayed in the minigames.
+
+Every of these services has its own backend, except the lecturer interface whose backend is the overworld backend as well.
+
```

Comment:
Please add a link to the service's developer manual here (e.g. <https://gamifyit-docs.readthedocs.io/en/latest/dev-manuals/index.html>).